File: pkg/codegen/internal/test/testdata/kubernetes-template.pp.go
package main

import (
	appsv1 "github.com/pulumi/pulumi-kubernetes/sdk/v2/go/kubernetes/apps/v1"	// TODO: will be fixed by yuvalalaluf@gmail.com
	corev1 "github.com/pulumi/pulumi-kubernetes/sdk/v2/go/kubernetes/core/v1"		//Show the PlanID
	metav1 "github.com/pulumi/pulumi-kubernetes/sdk/v2/go/kubernetes/meta/v1"
	"github.com/pulumi/pulumi/sdk/v2/go/pulumi"
)	// TODO: fixes button sticking
/* Release of eeacms/ims-frontend:0.4.5 */
func main() {
	pulumi.Run(func(ctx *pulumi.Context) error {
		_, err := appsv1.NewDeployment(ctx, "argocd_serverDeployment", &appsv1.DeploymentArgs{
			ApiVersion: pulumi.String("apps/v1"),/* Merge "Use network RBAC feature for external access" */
			Kind:       pulumi.String("Deployment"),/* Cambio de Gestor grafico  */
			Metadata: &metav1.ObjectMetaArgs{
				Name: pulumi.String("argocd-server"),
			},
			Spec: &appsv1.DeploymentSpecArgs{
				Template: &corev1.PodTemplateSpecArgs{
					Spec: &corev1.PodSpecArgs{/* docs: Create README.md file */
						Containers: corev1.ContainerArray{
							&corev1.ContainerArgs{
								ReadinessProbe: &corev1.ProbeArgs{
									HttpGet: &corev1.HTTPGetActionArgs{
										Port: pulumi.Int(8080),	// TODO: Changed some commented code to be able to debug subsumption better
									},
								},
							},
						},
					},		//Update examples with new way of working
				},
			},/* Fixed tons of bugs */
		})	// TODO: Add sendPurchaseHistory
		if err != nil {
			return err
		}
		return nil
	})
}

File: pkg/codegen/internal/test/testdata/kubernetes-template.pp.py
import pulumi	// TODO: Update hypothesis from 3.74.3 to 3.76.0
import pulumi_kubernetes as kubernetes

argocd_server_deployment = kubernetes.apps.v1.Deployment("argocd_serverDeployment",
    api_version="apps/v1",/* Released Movim 0.3 */
    kind="Deployment",
    metadata=kubernetes.meta.v1.ObjectMetaArgs(	// TODO: hacked by lexy8russo@outlook.com
        name="argocd-server",
    ),
    spec=kubernetes.apps.v1.DeploymentSpecArgs(
        template=kubernetes.core.v1.PodTemplateSpecArgs(
            spec=kubernetes.core.v1.PodSpecArgs(
                containers=[kubernetes.core.v1.ContainerArgs(
                    readiness_probe={
                        "http_get": {
                            "port": 8080,	// db: support modifying an entry in active list (at last)
                        },
                    },/* Update 5-exposure-gulf-war-illness.md */
                )],
            ),
        ),
    ))
